### ISSUE TITLE
fortran_fixed-form does not highlight "dimension" syntax

### ISSUE THREAD

**Issue by K1-ZR** [krvajal/vscode-fortran-support#127]
When `Implicit real*8(a-h,o-z)` is declared, `dimension` syntax can be used to define an array's dimension:

```fortran
DIMENSION myArray1(<size1>, <size2>, <size3>, ... ), myArray2(<size1>, <size2>, <size3>, ... ), ...
```

But fortran_fixed-form does not highlight this syntax.

**Comment by pedro-ricardo** [comment]
I'm not good at Fortran 77 style so help me out...
![Screenshot_20190415_102000](https://user-images.githubusercontent.com/15893711/56135960-2ac06480-5f68-11e9-8a94-b8e47b885cf2.png)
What you want is that `DIMENSION` is highlighted like `INTEGER` ?

**Comment by K1-ZR** [comment]
Yes. This is how it should look like (from visual studio):
![image](https://user-images.githubusercontent.com/44442133/56144327-d0bf9f00-5f67-11e9-9978-b1bea1fe1141.png)

**Comment by pedro-ricardo** [comment]
Fixed in PR #129 
:+1:
